src/components/function/getUserNameByUserId.ts

```diff
@@ -0,0 +1,21 @@
+import { Todo, User } from '../types/types';
+
+export function getUserNameByUserId(
+  userId: number,
+  todos: Todo[],
+  users: User[],
+): User | undefined {
+  const userTodo: Todo | undefined = todos.find(
+    (todo) => todo.userId === userId,
+  );
+
+  if (!userTodo) {
+    return undefined;
```

Comment:
```suggestion
    return null;
```
when we need to render nothing, it should be returned null.
undifined also render nothing, but general practice return null.